### PR TITLE
Documented render_variable function in prompt.py

### PR DESCRIPTION
**Pull request by pydanny**
Another bit of documentation. This is the first time I've ever touched this module!

### CODE REVIEW

File: cookiecutter/prompt.py [hunk 1]
```diff
@@ -85,6 +85,18 @@ def read_user_choice(var_name, options):
 
 
 def render_variable(env, raw, cookiecutter_dict):
+    """Inside the prompting taken from the cookiecutter.json file, this renders
+    the next variable. For example, if a project_name is "Peanut Butter Cookie", the repo_name could be be rendered with:
```
Comment (hackebrot) [inline cookiecutter/prompt.py]: This line needs to be wrapped 😃

File: cookiecutter/prompt.py [hunk 2]
```diff
@@ -85,6 +85,18 @@ def read_user_choice(var_name, options):
...
+
+        `{{ cookiecutter.project_name.replace(" ", "_") }}`.
+
+    This is then presented to the user as the default.
+
+    :param Environment env: A Jinja2 Environment object.
+    :param str raw: The next value to be prompted for by the user.
+    :param dict cookiecutter_dict: The deserialized cookiecutter.json file.
```
Comment (hackebrot) [inline cookiecutter/prompt.py]: Hmmm I'm afraid it's rather a Python dict that is populated in ``prompt_for_config`` based on "The deserialized cookiecutter.json file" and user input, if that makes sense. In the beginning it is just an empty dict and looks more and more like the json when looping for the key, value pairs.

**Comment by michaeljoseph** [comment]
:bell: @pydanny

**Comment by hackebrot** [comment]
Resolved the flake8 issue and adapted the docs for one of the parameters. 😄

**Pull request merged**